astro/src/content/blog/quantum-computing.mdx

```diff
@@ -0,0 +1,58 @@
+---
+publish_date: 2024-04-01
+title: "FusionAuth Cracks the Code: Quantum Passwords and a Universe of Secure Logins"
+description: How FusionAuth has built the world's first Quantum Vault
+authors: Alex Patterson
+image: /img/blogs/2023-hacktoberfest/hacktoberfest-2023.png
+categories: News
+tags: community story, april fools
+excerpt_separator: "{/* more */}"
+---
+Ever since the show [Devs](https://www.wired.com/story/inside-devs-dreamy-silicon-valley-quantum-thriller/) came out on FX in 2020, FusionAuth’s CEO Brian Pontarelli started thinking about the possibilities of adding quantum computing as part of FusionAuth’s product offering. We added a Skunkworks-style team that went into deep secrecy surrounding the project. After four years of iterating on both the physical design and the software to accompany the solution we're thrilled to announce a revolutionary new advancement in the realm of user authentication: **The FusionAuth Quantum Vault**. 
```

Comment:
```suggestion
Ever since the show [Devs](https://www.wired.com/story/inside-devs-dreamy-silicon-valley-quantum-thriller/) came out on FX in 2020, FusionAuth’s CEO Brian Pontarelli has been musing on the possibilities of adding quantum computing to FusionAuth. Four years ago, FusionAuth commissioned a Skunkworks that went into deep secrecy and worked on this concept. After years of iterating on both the physical design and the software to accompany the solution we're thrilled to announce a revolutionary new advancement in the realm of user authentication: **The FusionAuth Quantum Vault**. 
```